### PR TITLE
added cors enable flag 

### PR DESCRIPTION
**Pull request by strangerism**
I have added a flag to enable cors

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM golang:1.8 AS build-env
 
-MAINTAINER Mei Akizuru
+LABEL maintainer="Massimo Virgilio <massimovirgilio@hotmail.com>"
```
Comment (mayth) [inline Dockerfile]: **I'll continue to maintain this repository and the docker image.**
Of course, you can change here in your repository (which is forked from this repo), but please do not create the pull-request including such change 😢

File: server.go [hunk 2]
```diff
@@ -21,14 +21,16 @@ type Server struct {
 	// MaxUploadSize limits the size of the uploaded content, specified with "byte".
 	MaxUploadSize int64
 	SecureToken   string
+	CorsEnable   string
```
Comment (mayth) [inline server.go]: this should be `bool`.

File: simple_upload_server.go [hunk 3]
```diff
@@ -20,6 +20,7 @@ func run(args []string) int {
 	maxUploadSize := flag.Int64("upload_limit", 5242880, "max size of uploaded file (byte)")
 	tokenFlag := flag.String("token", "", "specify the security token (it is automatically generated if empty)")
 	logLevelFlag := flag.String("loglevel", "info", "logging level")
+	corsEnable := flag.String("cors_enable", "false", "set to true to enable cors")
```
Comment (mayth) [inline simple_upload_server.go]: use [`flag.Bool`](https://golang.org/pkg/flag/#Bool)

File: Dockerfile (hunk 1 above)
Comment (strangerism) [reply Dockerfile]: I am sorry, it's the first time I do pull request. I didn't know I wasn't supposed to change the maintainer on my forked repo. Just take the flag change, at the end I only wanted to share some good stuff. Also it was the first time I saw golang, hope I did it right.

File: simple_upload_server.go (hunk 3 above)
Comment (strangerism) [reply simple_upload_server.go]: I will do the changes as suggested and I'll try again. thanks

**Pull request closed without merging**